router/core/context.go

```diff
@@ -377,6 +379,10 @@ func (o *operationContext) Variables() []byte {
 	return o.variables
 }
 
+func (o *operationContext) Files() []httpclient.File {
```

Comment:
We export this interface also to module developers. Just as note. In future, we should expose it differently.